gc/2021/GC-04-20.md

```diff
@@ -0,0 +1,105 @@
+![WebAssembly logo](/images/WebAssembly.png)
+
+## Agenda for the April 20 video call of WebAssembly's Garbage Collection Subgroup
+
+- **Where**: zoom.us
+- **When**: April 20, 4pm-5pm UTC (April 6, 9am-10am Pacific Daylight Time)
+- **Location**: *link on calendar invite*
+
+### Registration
+
+None required if you've attended before. Fill out the form here to sign up if
+it's your first time: https://forms.gle/JehrAB4gWbtHjybt9. The meeting is open
+to CG members only.
+
+## Logistics
+
+The meeting will be on a zoom.us video conference.
+Installation is required, see the calendar invite.
+
+## Agenda items
+
+1. Opening, welcome and roll call
+    1. Opening of the meeting
+    1. Introduction of attendees
+1. Find volunteers for note taking (acting chair to volunteer)
+1. Adoption of the agenda
+1. Proposals and discussions
+    1. Discussion: Equirecursive Types in Binaryen [Thomas Lively] (1 hr)
+1. Closure
+
+## Meeting Notes
+
+### Introduction of attendees
+
+- Thomas Lively
+- Francis McCabe
+- Sergey Rubanov
+- Sabine
+- Ben Titzer
+- Alon Zakai
+- Lars Hansen
+- Slava Kuzmich
+- Derek Schuff
+- Igor Sheludko
+- Fil Pizlo
+- Keith Miller
+- Jakob Kummerow
+- Zalim Bashorov
+- Ross Tate
+- Aske
```

Comment:
Indeed only my first name was displayed on Zoom when you took attendance. For the notes, my full name is probably more appropriate.